### DISCUSSION TITLE
Is it possible to have a different name for the page title and the navbar?

### DISCUSSION THREAD

**Discussion by kakarj** [fuma-nama/fumadocs#1079] [Q&A] [answered]
<img width="846" alt="Screenshot 2024-11-20 at 9 31 11 AM" src="https://github.com/user-attachments/assets/4182b6d2-8c8e-4745-965c-dfaaf6051254">

**Comment by fuma-nama** [comment, marked as answer]
The children of page title component in page.tsx is customisable, you can either hardcode it, add a custom frontmatter property and use it as children, or simply remove them and add the title in MDX file instead. The default frontmatter title property is always used for metadata and sidebar

**Comment by Gitstar-OC** [comment]
Someone already has asked this, try searching for it.
